deploy/service.yaml

```diff
@@ -0,0 +1,186 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Docs service template for thethingsindustries.com
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterLabels:
+      CPU:
+        default: CPU limit of a container
+      DockerImage:
+        default: Docker image
+      Memory:
+        default: Memory limit of a container
+      TaskDesiredCount:
+        default: Number of container instances to run (prod only)
+Parameters:
+  CPU:
+    AllowedPattern: .+
```

Comment:
This can't just take anything. It needs to be a number, and considering we're on FARGATE, there are only a couple of possibilities.